### DISCUSSION TITLE
Anyone got this working with @sls-next/serverless-component?

### DISCUSSION THREAD

**Discussion by mohammedhammoud** [vinissimus/next-translate#441] [Q&A]
Have anyone got this working with [@sls-next/serverless-component](https://github.com/serverless-nextjs/serverless-next.js)?

**Comment by aralroca** [comment]
@mohammedhammoud did you get a similar error reported here: https://github.com/vinissimus/next-translate/issues/439 ? if not, it would be very helpful to know what error gives you 🙏
